test/emqx_web_hook_SUITE.erl

```diff
@@ -47,33 +47,22 @@ init_per_group(Name, Config) ->
     set_special_cfgs(),
     case Name of
         http ->
-            http_server:start_http(),
             emqx_ct_helpers:start_apps([emqx_web_hook], fun set_special_configs_http/1);
         https ->
-            http_server:start_https(),
             emqx_ct_helpers:start_apps([emqx_web_hook], fun set_special_configs_https/1);
         ipv6http ->
-            http_server:start_http(),
             emqx_ct_helpers:start_apps([emqx_web_hook], fun set_special_configs_ipv6_http/1);
         ipv6https ->
-            http_server:start_https(),
             emqx_ct_helpers:start_apps([emqx_web_hook], fun set_special_configs_ipv6_https/1)
     end,
-    io:format(standard_error, "Configs: ~p~n", [application:get_all_env(emqx_web_hook)]),
     Config.
 
-end_per_group(Name, Config) ->
-    case lists:member(Name,[http, ipv6http]) of
-        true ->
-            http_server:stop_http();
-        _ ->
-            http_server:stop_https()
-    end,
+end_per_group(_Name, Config) ->
     emqx_ct_helpers:stop_apps([emqx_web_hook]),
     Config.
 
 set_special_configs_http(_) ->
-    ok.
+    application:set_env(emqx_web_hook, url, "http://127.0.0.1:9999").
```

Comment:
Why do we need to modify this http port?

Reply:
8081 used for `http:management`